components/brave_wallet/browser/bitcoin/bitcoin_discover_account_task.h

```diff
@@ -33,6 +35,8 @@ struct DiscoveredBitcoinAccount {
 
 class DiscoverAccountTaskBase {
  public:
+  enum class DiscoveryStage : uint32_t { kReceive, kChange };
```

Comment:
I think this can be just:

```cxx
enum class DiscoveryStage { kReceive, kChange };
```

Reply:
this is for to_underlying to return uint32_t

Reply:
But does it matter? `to_underlying` will deduce a type anyway, and if we are not doing anything that particularly warrants `uint32_t`, we should just let it be on its defaults.